app/models/rdv.rb

```diff
@@ -77,6 +77,18 @@ def in_the_future?
     starts_at > Time.zone.now
   end
 
+  def in_the_past?
+    starts_at <= Time.zone.now
+  end
+
+  def in_one_hour?
```

Comment:
suggestion de nommage, qu’en penses-tu ? `in_next_hour?`
```suggestion
  def in_next_hour?
```